lib/cisco_node_utils/vlan.rb

```diff
@@ -206,5 +224,86 @@ def mapped_vni=(vni)
     def default_mapped_vni
       config_get_default('vlan', 'mapped_vni')
     end
+
+    # Helper method to delete @set_args hash keys
+    def set_args_keys_default
+      @set_args = {}
+    end
+
+    def private_vlan_type
+      result = config_get('vlan', 'private_vlan_type_all')
+      return '' if result.nil?
+      result.each do |elem|
+        vlan = elem.match(@vlan_id)
+        next if vlan.nil?
+        return elem.match(/\d+\s+(\S+)/)[1]
+      end
+      ''
+    end
+
+    def private_vlan_type=(pv_type)
+      Feature.private_vlan_enable
+      fail TypeError unless pv_type && pv_type.is_a?(String)
+      set_args_keys_default
+      if pv_type == default_private_vlan_type
+        pv_type = private_vlan_type
```

Comment:
Not sure I understood. This is how I differentiate between 'no' command and set command.
How exactly is it already default?